src/Components/NavBar/NavBar.js

```diff
@@ -12,7 +12,9 @@ export const NavBar = () => {
 
   return (
     <div className='navbar'>
-      <button className='navbtn'>Options</button>
+      <Link to='/settings' >
```

Comment:
It's effectively the same thing but less code because it only has to check one thing. It doesn't matter where your navigating from or going to, it doesn't matter if you enter a url directly, it just runs the same block every time it renders the navabr.